### ISSUE TITLE
DBeaver 21.0.0 does not read certs from MacOS Keychains as it did in prior releases

### ISSUE THREAD

**Issue by els-pnw** [dbeaver/dbeaver#11579]
#### System information: 
- Operating system (distribution) and version
- MacOS 10.15.7
- DBeaver version
- 21.0.0
- Additional extensions

#### Connection specification:
- Database name and version
- n/a
- Driver name
- jdbc teiid driver (doesn't really matter)
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
- No

#### Describe the problem you're observing:
In prior releases Dbeaver would read / use internal certs registered with MacOS Keychains, this no longer works.
#### Steps to reproduce, if exist:
Attempt to connect to a database that uses an internal cert that would typically need to be installed in either a Java security/certs file or to MacOS's keychain.

#### Include any warning/errors/backtraces from the logs

TEIID20020 Error establishing socket to host and port:  host.com:31000. Reason: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
  PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
  PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
    unable to find valid certification path to requested target
    unable to find valid certification path to requested target

**Comment by serge-rider** [comment]
Most likely it happens because now we include Java in DBeaver package. So system-default Java not used anymore.

**Comment by yanliuy-max** [comment]
版本21.0.1.202103211229,unable to find valid certification path to requested target when connect the db2 in win10 
also cann't find the sslconnection  setting

**Comment by els-pnw** [comment]
@serge-rider that's correct, the workaround in the past was to install the pkg instead of the dmg; however, pkg is no longer available.  

The other issue here is that I must use keytool to add the cert after every upgrade.  Is there no way to package this up such that the Java security/cacerts is NOT overwritten.  This seems like something that should be doable and this is the root of the issue, with each install the entire DBeaver.app folder is overwritten, including cacerts.

**Comment by serge-rider** [comment]
Unfortunately it is not possible to patch or backup any files using DMG installer.
At the moment we don't have plans to return PKG installer (as it requires root permissions and more restrictive notarization process).

**Comment by serge-rider** [comment]
We plan to add dmg installer without bundled Java in 21.2 version.

**Comment by Matvey16** [comment]
#5704

**Comment by Matvey16** [comment]
@els-pnw  Please check our EA version https://dbeaver.io/files/ea/ it should have been fixed in it

**Comment by madhsiva** [comment]
Is there any update on this. Getting this error in all versions. any help
